content/english/chatbot/activity-3.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Activity 3: Add your own categories"
+draft: false
+weight: 7
+---
+
+## Add more categories
+
+Right now, our bot can only respond to "Hello". Let's add some more categories so that our bot can respond to more things.
+
+1. If the user says "What is your name?" the bot should respond with "My name is Chatbot".
+2. If the user says "What is your favorite color?" the bot should respond with "My favorite color is blue".
+3. If the user says "What is your favorite food?" the bot should respond with "My favorite food is pizza".
+
+As a reminder, here is the basic structure of a category:
+
+```
+<category>
+    <pattern>HELLO</pattern>
+    <template>
+        Hello, World!
+    </template>
+</category>
+```
+
+{{% notice tip %}}

```

Comment:
Also as a reminder/tip to user - to save the file whenever they modify the categories and want to interact with the Chatbot again with the updated modifications.